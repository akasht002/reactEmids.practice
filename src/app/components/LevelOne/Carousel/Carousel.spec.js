import React from 'react'
import {Carousel} from './index'
import renderer from 'react-test-renderer'


test('Carousel Component Testing', () => {
    const  ServiceTypeSettings = {
        dots: false,
        infinite: false,
        speed: 500,
        slidesToShow: 2,
        slidesToScroll: 2,
        variableWidth: true
    }
  const component = renderer.create(<Carousel {...ServiceTypeSettings}/>)
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
