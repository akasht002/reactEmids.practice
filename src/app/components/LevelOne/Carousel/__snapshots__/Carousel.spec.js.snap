// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Carousel Component Testing 1`] = `
<div
  className="slick-slider slick-initialized"
>
  
  <div
    className="slick-list"
  >
    <div
      className="slick-track"
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onMouseOver={[Function]}
      style={
        Object {
          "WebkitTransform": "translate3d(0px, 0px, 0px)",
          "WebkitTransition": "",
          "msTransform": "translateX(0px)",
          "opacity": 1,
          "transform": "translate3d(0px, 0px, 0px)",
          "transition": "",
        }
      }
    />
  </div>
  
  
</div>
`;
